### ISSUE TITLE
Error in while (term == 1) sys.sleep(10) : the condition has length > 1

### ISSUE THREAD

**Issue by JiaxinWang123** [aestovall/TLSLeAF#2]
Hi Atticus Stovall,

I was trying to use TLSLeAF but I was stuck by the error when ran:

TLSLeAF.df<-TLSLeAF(input_file,
                    overwrite=FALSE,
                    center,
                    scatterLim=80,
                    SS=0.02,
                    scales=c(0.1,0.5,0.75),
                    rf_model=rf_model,
                    correct.topography = TRUE,
                    voxRes=0.1,
                    minVoxDensity=5,
                    superDF=TRUE,
                    clean=TRUE,
                    OS='windows')

Here is the error:
Error in while (term == 1) sys.sleep(10) : the condition has length > 1
In addition: Warning message:
In system(cmd, intern = intern, wait = wait | intern, show.output.on.console = wait,  :
  running command 'C:\WINDOWS\system32\cmd.exe /c cloudcompare -SILENT -C_EXPORT_FMT ASC -PREC 6 -NO_TIMESTAMP -AUTO_SAVE OFF -COMPUTE_NORMALS -O MYPATH\0628_summer.ptx -SAVE_CLOUDS FILE MYPATH\0628_summer.asc -CLEAR' had status 1

Please let me know if you have any ideas about that.
Thanks!

**Comment by UQ-Raja-Ram-Aryal** [comment]
Hi JiaxinWang123,

Is the above-mentioned problem solved? I am also getting the same issue.

**Comment by JiaxinWang123** [comment]
> Hi JiaxinWang123,
> 
> Is the above-mentioned problem solved? I am also getting the same issue.

No, I ended up with developing my own pipeline, and still working on it.

**Comment by UQ-Raja-Ram-Aryal** [comment]
Thank you for your quick response. Let me know If you get it.
